app/controllers/api/v1/team_users_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    # TeamUsersController
+    class TeamUsersController < ApplicationController
+      def index
+        @team_users = team.team_users
+
+        render json: @team_users, each_serializer: TeamUserSerializer
+      end
+
+      def show
+        if team_user
+          render json: team_user, serializer: TeamUserSerializer
+        else
+          render json: team_user.errors
```

Comment:
what will happen if team_user is not found?

Reply:
else branch of show action never run because find-error catch with :record_not_found hook

Reply:
fixed